core/src/main/java/org/polypheny/db/rex/RexBuilder.java

```diff
@@ -1426,10 +1423,7 @@ private static String padRight( String s, int length ) {
         if ( s.length() >= length ) {
             return s;
         }
-        return new StringBuilder()
-                .append( s )
-                .append( Spaces.MAX, s.length(), length )
-                .toString();
+        return StringUtils.rightPad( s, length - s.length() );
```

Comment:
`.append( Spaces.MAX, s.length(), length )` appends `length - s.length()` spaces, but `rightPad` takes the length the string should be after padding, so it should be
```java
return StringUtils.rightPad( s, length );
```
right?

Reply:
The `if` is then not needed as well.